docs/en/introduction.md

```diff
@@ -0,0 +1,99 @@
+# Introduction
+
+> In this chapter, we will use the Ribir syntax to write some simple examples. You only need to understand the general idea, no need to go into details. We will introduce it in detail in the following chapters.
+
+## What is Ribir?
+
+Ribir is a Rust framework for building cross-platform user interfaces. It allows you to compile applications for desktop, mobile and browser with one code. Ribir uses a non-intrusive declarative programming model that allows you to develop and design user interfaces as an independent module.
```

Comment:
Ribir is an open source Rust framework for building beautiful, natively compiled, multi-platform applications from a single codebase.